### PR TITLE
Add flake8 linting for Python

### PR DESCRIPTION
**Pull request by SuperQ**
Add flake8 CircleCI to lint Python scripts.
* Add TODO tw_cli.py, not Python3 compatible.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
CC @dswarbrick

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[flake8]
```
Comment (brian-brazil) [inline .flake8:1]: We should probably be consistent for this line: https://github.com/prometheus/client_python/blob/master/tox.ini#L83

Comment (SuperQ) [reply .flake8:1]: Makes sense, done.

**Comment by dswarbrick** [comment]
Nice, I am hugely in favour of this. We do some pretty strict Python linting where I work, including https://github.com/psf/black, which kinda brings Python code into the world of `go fmt`.

btw, I can prepare a PR for tw_cli.py if desired - we apply a debian patch to this in the debian package, to make it python3 compatible.

**Comment by SuperQ** [comment]
Yes, it would be appreciated if you could update tw_cli.py.

**Comment by dswarbrick** [comment]
> Yes, it would be appreciated if you could update tw_cli.py.

See #80

**Pull request merged**